content/docs/source-git/how-to-source-git.md

```diff
@@ -5,13 +5,179 @@ draft: false
 weight: 2
 ---
 
+# How to create a source-git repository?
+
+This guide walks through the steps to create a source-git repository from an
+upstream project.
+
+Please bare in mind, that the structure of source-git repositories is still
+being discussed and in development. Due to this, this guide is subject to
+change. Consider joining the Fedora Source-git SIG if you are interested and
+would like to participate in the discussions.
+
+The process to construct a source-git repository and a branch to track
+downstream (distribution) work, which then can be synced to dist-git has the
+following steps:
+
+1. Identify the upstream Git repository and corresponding to the current version
+   of the software, as available in the distribution.
+2. Create and populate a `.distro` subdirectory in the upstream working tree
+   to hold the files needed to package and test the software in the
+   distribution.
+3. Configure syncing to dist-git in `.distro/source-git.yaml`.
+4. Remove the references to the downstream (distribution) patches from the
+   spec-file and apply these patches as Git-commits.
+5. Test the source-git repository and branch by trying to sync the content to
```

Comment:
```suggestion
5. Test the source-git repository branch by trying to sync the content to
```

Reply:
You marked this resolved without no action :-)

I don't understand the "source-git repository AND branch", that's why I suggested removing the "and", but given that your last section below is now called `## Test the source-git repository` I'd now suggest removing the "and branch"

Reply:
Sorry, I think I saw this diff in the reverse. I blame being tired.

There is "and branch" here, b/c the process is really about setting up a source-git repo and a branch to track a distribution version. Something like this is already written a few lines above: "The process to construct a source-git repository _and a branch_ to track downstream (distribution) work..."

But yes, I agree that despite this, reading this sentence can be confusing. So maybe it's better to be a little bit less correct and keep things simple. Removed "and branch".